openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/NodeSuggestions.component.tsx

```diff
@@ -61,6 +61,25 @@ const NodeSuggestions: FC<EntitySuggestionProps> = ({
     }
   };
 
+  const getSuggestResults = async (value: string) => {
+    try {
+      const data = await getSuggestions(
+        value,
+        SearchIndex[
+          entityType as keyof typeof SearchIndex
+        ] as ExploreSearchIndex
```

Comment:
```suggestion
        SearchIndex[
          entityType as keyof typeof SearchIndex
        ]
```

Reply:
If we removed `as ExploreSearchIndex` it will throw a warning!